_posts/2017-06-28-Openshift-an-introduction.md

```diff
@@ -0,0 +1,241 @@
+
+__Deployment automation:__
+OpenShift automated the deployment of application containers. It supports rolling deployments for multi-containers apps and allows you to roll back to an older version.
+
+__Continuous integration:__
+It provides built-on continuous integration capabilities with Jenkins and can also tie into your existing CI solutions.The OpenShift’s Jenkins image can also be used to run your Jenkins masters and slaves on openshift.
+
+### Scale
+<span class="image left small"><img class="p-image" alt="scale" src="/img/openshift-an introduction/scale.png"></span>
+When you want to start scaling your application wheter it's from 1 replica to 2 or scale it to 2000 replicas a lot of complexity is added.
+OpenShift leverages the power of containers and a incredibly powerful orchestration engine to make that happen. Containers make sure that applications are packed up in their own space and are independent from the OS, this makes applications incredibly portable and hyper scalable. OpenShift’s orchestration layer, Google’s Kubernetes, automates the scheduling and replication of these containers meaning that they’re highly available and able to accommodate whatever your users can throw at it.
+This means that your team spends less time in the weeds and keeping the lights on, and more time being innovative and productive. 
+
+### Opensource
+<span class="image left small"><img class="p-image" alt="opensource" src="/img/openshift-an introduction/opensource.png"></span>
+There are multiple versions of openshift (Spoiler: it's gonna be the next topic in this blog post) but they are all based on Openshift Origin.
+Origin provides an open source application container platform. All source code for the Origin project is available under the Apache License (Version 2.0) on [GitHub](https://github.com/openshift)
+
+# Openshift landscape
+
+There are a few different openshift releases depending what you need.
+As of this writing the openshift landscape looks like this.
+
+<p>
+  <img class="image fit" alt="openshiftlandscape" src="/img/openshift-an introduction/openshiftlandscape.png">
+</p>
+
+### Openshift origin
+It's the upstream community project used in openshift Online, Openshift dedicated and Openshift container Platform.
+It's build around docker and kubernetes cluster management.
+Origin is augmented by application lifecycle management functionality and DevOps tooling.
+Origin updates as often as open source developers contribute via git sometimes as often as several times per week. Here you get the new feature the quickest but at the cost of stability.
+
+### Openshift container platform
+Formaly known as openshift enterprise.
+It's the  platform software  to deploy and manage openshift on your own  infrastructure of choice.
```

Comment:
It's the platform software
Double space after "the".
OpenShift
Also a double space after "your own"